ts/parts-3d/Column.ts

```diff
@@ -223,15 +218,33 @@ seriesTypes.column.prototype.translate3dShapes = function (): void {
             (shapeArgs as any).depth = depth;
             (shapeArgs as any).insidePlotArea = true;
 
+            // Point's position in 2D
+            point2dPos = {
+                x: (shapeArgs as any).x + (shapeArgs as any).width / 2,
+                y: (shapeArgs as any).y,
+                z: z + depth / 2 // The center of column in Z dimension
+            };
+
+            // Recalculate point positions for inverted graphs
+            if (chart.inverted) {
+                point2dPos.x = (shapeArgs as any).height;
+                point2dPos.y = point.clientX;
+            }
+
+            // Calculate and store point's position in 3D,
+            // using perspective method.
+            point.pos3d = perspective([point2dPos], chart, true, false)[0];
```

Comment:
Just nitpicking: since `pos3d` is equivalent to plotX/Y then maybe `plot3d` name would be better?

Reply:
I have changed this name in 65ed49e